app/extras/user-status-map.html

```diff
@@ -0,0 +1,30 @@
+<table class="table table-condenced">
+    <thead>
+        <tr>
+            <th>ID</th><th>Name</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>109000100</td><td>Visitor</td>
+        </tr>
+        <tr>
+            <td>109000200</td><td>Active</td>
+        </tr>
+        <tr>
+            <td>109000300</td><td>LoggedIn</td>
+        </tr>
+        <tr>
+            <td>109000400</td><td>Inactive</td>
+        </tr>
+        <tr>
+            <td>109000500</td><td>Banned</td>
+        </tr>
+        <tr>
+            <td>109000600</td><td>Removable</td>
+        </tr>
+        <tr>
+            <td>109000700</td><td>Archived</td>
+        </tr>
+    </tbody>
+</table>
```

Comment:
add new line

Reply:
Renamed the file to combine both user and course statuses. Added a new line at the end.